deploy/kubernetes/setup-upcloud-csi.yaml

```diff
@@ -220,6 +220,16 @@ spec:
       priorityClassName: system-node-critical
       serviceAccount: csi-upcloud-node-sa
       hostNetwork: true
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
```

Comment:
Should we use double quotes with these values?

Reply:
Never mind my previous comment. It seems that our spec is not that consistent with these as I first thought.

Reply:
Well, I also not 100% sure what is the standard, but in majority of the cases we are not using quotes